texar/torch/data/tokenizers/pretrained_distilbert_tokenizer.py

```diff
@@ -0,0 +1,165 @@
+            'distilbert-base-uncased-distilled-squad-config.json',
+            _DISTILBERT_PATH + 'bert-large-uncased-vocab.txt',
+        ],
+    }
+    _MAX_INPUT_SIZE = {
+        'distilbert-base-uncased': 512,
+        'distilbert-base-uncased-distilled-squad': 512,
+    }
+    _VOCAB_FILE_MAP = {
+        'vocab_file': {
+            'distilbert-base-uncased': 'bert-base-uncased-vocab.txt',
+            'distilbert-base-uncased-distilled-squad':
+                'bert-large-uncased-vocab.txt',
+        }
+    }
+
+    @staticmethod
+    def default_hparams():
+        r"""Returns a dictionary of hyperparameters with default values.
+
+        * The tokenizer is determined by the constructor argument
+          :attr:`pretrained_model_name` if it's specified. In this case,
+          `hparams` are ignored.
+        * Otherwise, the tokenizer is determined by
+          `hparams['pretrained_model_name']` if it's specified. All other
+          configurations in `hparams` are ignored.
+        * If the above two are `None`, the tokenizer is defined by the
+          configurations in `hparams`.
+
+        .. code-block:: python
+
+            {
+                "pretrained_model_name": "distilbert-base-uncased",
+                "vocab_file": None,
+                "max_len": 512,
+                "unk_token": "[UNK]",
+                "sep_token": "[SEP]",
+                "pad_token": "[PAD]",
+                "cls_token": "[CLS]",
+                "mask_token": "[MASK]",
+                "tokenize_chinese_chars": True,
+                "do_lower_case": True,
+                "do_basic_tokenize": True,
+                "non_split_tokens": None,
+                "name": "distilbert_tokenizer",
+            }
+
+        Here:
+
+        `"pretrained_model_name"`: str or None
+            The name of the pre-trained DistilBERT model.
+
+        `"vocab_file"`: str or None
+            The path to a one-wordpiece-per-line vocabulary file.
+
+        `"max_len"`: int
+            The maximum sequence length that this model might ever be used with.
+
+        `"unk_token"`: str
+            Unknown token.
+
+        `"sep_token"`: str
+            Separation token.
+
+        `"pad_token"`: str
+            Padding token.
+
+        `"cls_token"`: str
+            Classification token.
+
+        `"mask_token"`: str
+            Masking token.
+
+        `"tokenize_chinese_chars"`: bool
+            Whether to tokenize Chinese characters.
+
+        `"do_lower_case"`: bool
+            Whether to lower case the input
+            Only has an effect when `do_basic_tokenize=True`
+
+        `"do_basic_tokenize"`: bool
+            Whether to do basic tokenization before wordpiece.
+
+        `"non_split_tokens"`: list
+            List of tokens which will never be split during tokenization.
+            Only has an effect when `do_basic_tokenize=True`
+
+        `"name"`: str
+            Name of the tokenizer.
+
+        """
+        return {
```

Comment:
Is `default_hparams` identical to `BertTokenizer`? If so, reuse by 
```
default_ = BertTokenzier.default_hparams()
default_['pretrained_model_name'] = 'distilbert-base-uncased'
return default_
```

If not, the statement in https://github.com/asyml/texar-pytorch/pull/201/files#diff-5910de50635db90a6aec9eb7ad7bbbc2R29 should be updated and point out which part is different